tools/tangle-cli/src/commands/vc/vcCommand.ts

```diff
@@ -5,22 +5,24 @@ import IssueVcCommand from "./issueVcCommand";
 import PresentVcCommand from "./presentVcCommand";
 import VerifyVcCommand from "./verifyVcCommand";
 
-const params: ICommandParam[] = [];
+const params: ICommandParam[] = [
+  {
```

Comment:
don't repeat this definition, it should be in one place, modulo the comments made before